### PR TITLE
Start Absinthe.Schema for persistent term schema

### PR DESCRIPTION
**Pull request by andersonmcook**
Locally we ran into an error generating a JSON schema file. We're using `Absinthe.Schema.PersistentTerm`

```elixir
@schema_provider Absinthe.Schema.PersistentTerm
```

Error:
```
mix absinthe.schema.json --schema OurSchema --pretty path/to/schema.json
** (ArgumentError) errors were found at the given arguments:

  * 1st argument: no persistent term stored with this key

    :persistent_term.get({Absinthe.Schema.PersistentTerm, OurSchema})
    (absinthe 1.6.6) lib/absinthe/schema/persistent_term.ex:94: Absinthe.Schema.PersistentTerm.get/1
    (absinthe 1.6.6) lib/absinthe/schema/persistent_term.ex:54: Absinthe.Schema.PersistentTerm.__absinthe_type__/2
    (absinthe 1.6.6) lib/absinthe/phase/schema.ex:107: Absinthe.Phase.Schema.set_schema_node/4
    (absinthe 1.6.6) lib/absinthe/phase/schema.ex:71: anonymous fn/4 in Absinthe.Phase.Schema.set_children/3
    (absinthe 1.6.6) lib/absinthe/blueprint/transform.ex:16: anonymous fn/3 in Absinthe.Blueprint.Transform.prewalk/2
    (absinthe 1.6.6) lib/absinthe/blueprint/transform.ex:109: Absinthe.Blueprint.Transform.walk/4
    (elixir 1.13.0) lib/enum.ex:1715: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
```

I was able to get it to generate by wrapping the task
```elixir

defmodule Mix.Tasks.Wrapper do
  use Mix.Task

  def run(argv) do
    children = [{Absinthe.Schema, OurSchema}]
    Supervisor.start_link(children, strategy: :one_for_one, name: Our.Supervisor)
    Mix.Tasks.Absinthe.Schema.Json.run(argv)
  end
end
```

I apologize if there was already a built-in or a better way to do this :)

### CODE REVIEW

File: lib/mix/tasks/absinthe.schema.json.ex [hunk 1]
```diff
@@ -80,6 +80,12 @@ defmodule Mix.Tasks.Absinthe.Schema.Json do
 
     opts = parse_options(argv)
 
+    # Start for Absinthe.Schema.PersistentTerm
```
Comment (kdawgwilk) [inline lib/mix/tasks/absinthe.schema.json.ex:83]: Would this also need to be added to `absinthe.schema.sdl`?

Comment (andersonmcook) [reply lib/mix/tasks/absinthe.schema.json.ex:83]: Great question! I'm unsure, we had no problem running `absinthe.schema.sdl` locally.  It was only when we tried `absinthe.schema.json`.

Comment (dolfinus) [reply lib/mix/tasks/absinthe.schema.json.ex:83]: Could you please also add the same change to `absinthe.schema.sql`? This could fix #1054

**Comment by andersonmcook** [comment]
Ah, I see there's CI for Elixir 1.10 but `tmp_dir` was introduced in 1.11.  I can work around that.

**Comment by benwilson512** [comment]
Thank you!

**Pull request merged**